Makefile

```diff
@@ -1,3 +1,11 @@
+bdt:
+	# Please set env vars BDT_CLIENT_ID and BDT_CLIENT_SECRET before running
```

Comment:
Can you provide an example on how to pass the env vars to `make`?  Something simple like we have [here](https://github.com/CMSgov/bcda-app/blob/master/Makefile#L2-L3).